src/content/App/App.tsx

```diff
@@ -28,6 +30,41 @@ function App() {
   // const prevSections = useRef(sections);
   // Sync initial state with chrome storage on mount
   useEffect(() => {
+    const fetchSecureToken = async () => {
```

Comment:
feel like this function could be moved to `utils.ts` with the other "fetch from workday API" functions. although, if there's something that makes that more complicated, don't worry about it.